internal/cli/cmd/cmd.go

```diff
@@ -88,20 +87,10 @@ cilium connectivity test`,
 		newCmdStatus(),
 		newCmdSysdump(hooks),
 		newCmdVersion(),
+		newCmdInstallWithHelm(),
+		newCmdUninstallWithHelm(),
+		newCmdUpgradeWithHelm(),
 	)
-	if utils.IsInHelmMode() {
```

Comment:
I think we need to update the README as well? https://github.com/cilium/cilium-cli?tab=readme-ov-file#helm-installation-mode

Reply:
yeah i was going to update README.md in a subsequent PR after removing the classic mode from all the commands but i don't mind updating it in this PR.

i'm pretty sure the classic mode is not being used in cilium/cilium CI anymore, but i'll run the CI to make sure 🚀 🙏